install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -1602,9 +1602,21 @@ localRedirectPolicy: false
 # To include or exclude matched resources from cilium identity evaluation
 # labels: ""
 
-# logOptions allows you to define logging options. eg:
-# logOptions:
-#   format: json
+# -- (string) Allows you to define logging options.
```

Comment:
Let's use a consistent style for the option description, “allows you to...”  is not really necessary here.

```suggestion
# -- (string) Define logging options.
```